website/app/components/options-form.gjs

```diff
@@ -0,0 +1,120 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { on } from '@ember/modifier';
+import { htmlSafe } from '@ember/template';
+
+export const UPLOAD_TYPES = {
+  simulated: 'Simulated 🧑‍🔬',
+  http: 'HTTP 📡',
+};
+
+// Simulated
+// Values in kilobits per second (kbps)
+const RATES = {
+  'Disconnected - 0 Mbps': 0,
+  'Very slow - 0.1 Mbps': 100,
+  'Slow 3G - 0.4 Mbps': 400,
+  'Fast 3G - 0.675 Mbps': 675,
+  'ADSL - 1.5 Mbps': 1_500,
+  '4G/LTE - 50 Mbps': 50_000,
+  'Fast Fibre - 100 Mbps': 100_000,
+};
+const DEFAULT_RATE = RATES['Fast 3G - 0.675 Mbps'];
+
+// HTTP
+const DEFAULT_URL = 'https://api.bytescale.com/v1/files/basic';
+const METHODS = ['GET', 'POST', 'PUT', 'PATCH', 'HEAD'];
+const DEFAULT_METHOD = 'POST';
+const DEFAULT_HEADERS = {
+  Authorization: 'Basic YXBpa2V5OmZyZWU',
+  'Content-Type': 'application/x-www-form-urlencoded',
+};
```

Comment:
While looking for a public "black hole" upload endpoint I found this documented in the MIT npm package `react-uploader` [README](https://www.npmjs.com/package/react-uploader#-file-management-api)

It seems to work for me, but it's a suggestion that user's may want to use. They are using this API on their own behalf 🤷🏻

Reply:
Ended up replacing this with a WireMock endpoint from a free account I created